apps/mobile/index.js

```diff
@@ -45,7 +48,7 @@ class App extends React.Component<Props, State> {
 
   render() {
     if (this.state.fontsLoaded) {
-      return <SharedApp />;
+      return STORYBOOK === 'true' ? <StorybookUI /> : <SharedApp />;
```

Comment:
This might not be great if we ever wanted to build and ship the app to an app store, as the Storybook would get bundled in it it too. But it's good enough for now.